virtwho/virt/kubevirt/kubevirt.py

```diff
@@ -84,14 +85,23 @@ def getHostGuestMapping(self):
             status = node['status']
             version = status['nodeInfo']['kubeletVersion']
             name = node['metadata']['name']
-            host_id = status['nodeInfo']['machineID']
-            address = status['addresses'][0]['address']
+
+            uuid = status['nodeInfo']['machineID']
```

Comment:
This should be the system uuid.

Reply:
Kubernetes guys suggest [1] to use `machineID` instead.

> SystemUUID reported by the node. For unique machine identification MachineID is preferred

[1] https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/#nodesysteminfo-v1-core

Reply:
For this purpose, yes.